include/bm/bm_sim/queueing.h

```diff
@@ -397,9 +397,9 @@ class QueueingLogicRL {
 //! high priority can starve lower-priority queues. For example, if the queue
 //! with priority `0` always contains at least one element, the other queues
 //! will never be served.
-//! As for QueueingLogicRL, the write behavior (push_front()) is blocking: once
-//! a logical queue is full, subsequent incoming elements will be dropped until
-//! the queue starts draining again.
+//! As for QueueingLogicRL, the write behavior (push_front()) is not blocking:
```

Comment:
Sorry, I do not know the BMv2 internals very well at all, so just throwing up a random red flag on comment changes -- After your changes is the phrase "As for QueueingLogicRL" still correct?

Reply:
I didn't change anything in this file. I realized this comment was not correct so I fixed it, but it has nothing to do with the rest of the changes.